services/authentication/src/main/java/entity/User.java

```diff
@@ -0,0 +1,35 @@
+package entity;
+
+/**
+ * Created by Michele on 12/07/2017.
+ */
+public class User {
+    private String username;
+    private String password;
+
+    public User(String username, String password) {
+        this.username = username;
+        this.password = password;
+    }
+
+    public User() {
```

Comment:
Ok, never mind. I deleted the default constructors thanks to the JSonProperty setting.